lib/PuppeteerSharp.TestServer/Program.cs

```diff
@@ -1,29 +1,19 @@
-﻿using System;
-using System.IO;
-using System.Net;
-using Microsoft.AspNetCore;
-using Microsoft.AspNetCore.Hosting;
-
+﻿using System.Net;
+using Microsoft.AspNetCore.Hosting;
+
 namespace PuppeteerSharp.TestServer
 {
-    class Program
+    public class Program
     {
-        static void Main(string[] args)
-        {
-            var host = WebHost.CreateDefaultBuilder(args)
-                .UseKestrel()
-                .UseStartup<Startup>()
-                .UseKestrel(options =>
+        public static IWebHostBuilder GetWebHostBuilder(params string[] args) => new WebHostBuilder()

```

Comment:
I think we don't need a "Program" class anymore. Or we rename it to WebServer.cs or we can move it to the Startup class.